src/java/picard/sam/GatherBamFiles.java

```diff
@@ -27,16 +27,28 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Concatenates one or more BAM files together as efficiently as possible. Assumes that the " +
-                "list of BAM files provided as INPUT are in the order that they should be concatenated and simply concatenates the bodies " +
-                "of the BAM files while retaining the header from the first file.  Operates via copying of the gzip blocks directly for speed " +
-                "but also supports generation of an MD5 on the output and indexing of the output BAM file. Only support BAM files, does not " +
-                "support SAM files.",
-        usageShort = "Concatenates one or more BAM files together as efficiently as possible",
+        usage = GatherBamFiles.USAGE_SUMMARY + GatherBamFiles.USAGE_DETAILS,
+        usageShort = GatherBamFiles.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class GatherBamFiles extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Concatenates one or more BAM files together as efficiently as possible";
+    static final String USAGE_DETAILS = "Program to perform a rapid \"gather\" operation on BAM files after a scatter" +
+            " operations where the same process has been performed on different regions of a BAM file creating many " +
+            "smaller BAM files that now need to be concatenated (reassembled) back together." +
+            "<br /><br />" +
+            "Assumes that the list of BAM files provided as INPUT are in the order that they should be concatenated and" +
+            " simply concatenates the bodies of the BAM files while retaining the header from the first file.  " +
+            "Operates via copying of the gzip blocks directly for speed but also supports generation of an MD5 on the" +
+            " output and indexing of the output BAM file. Only support BAM files, does not support SAM files." +
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar GatherBamFiles \\<br /> " +
+            "     I=myBAM.bam \\ <br /> " +
```

Comment:
This example should specify more than one input BAM file, since that's the use case for which this program is intended.